File: comments_server.py
# Python Flask backend for comments
from flask import Flask, request, jsonify
from flask_cors import CORS
app = Flask(__name__)
CORS(app)
comments = []

@app.route('/api/comments', methods=['GET'])
def get_comments():
    return jsonify(comments)

@app.route('/api/comments', methods=['POST'])
def post_comment():
    data = request.get_json()
    name = data.get('name')
    comment = data.get('comment')
    if name and comment:
        comments.append({'name': name, 'comment': comment, 'date': str(datetime.datetime.now())})
        return jsonify({'success': True}), 201
    return jsonify({'success': False, 'error': 'Name and comment required.'}), 400

if __name__ == '__main__':
    app.run(port=4000)

File: comments_server.js
// Node.js Express backend for comments
const express = require('express');
const cors = require('cors');
const bodyParser = require('body-parser');
const app = express();
const PORT = 4000;

app.use(cors());
app.use(bodyParser.json());

let comments = [];

app.get('/api/comments', (req, res) => {
  res.json(comments);
});

app.post('/api/comments', (req, res) => {
  const { name, comment } = req.body;
  if (name && comment) {
    comments.push({ name, comment, date: new Date().toISOString() });
    res.status(201).json({ success: true });
  } else {
    res.status(400).json({ success: false, error: 'Name and comment required.' });
  }
});

app.listen(PORT, () => {
  console.log(`Comments server running on http://localhost:${PORT}`);
});
